src/uattributes.rs

```diff
@@ -95,15 +99,14 @@ impl UAttributes {
         reply_to_address: UUri,
         options: CallOptions,
     ) -> Self {
-        let ttl = i32::try_from(options.timeout()).unwrap_or(i32::MAX);
         Self {
             type_: UMessageType::UMESSAGE_TYPE_REQUEST.into(),
             id: Some(message_id).into(),
             priority: UPriority::UPRIORITY_CS4.into(),
             source: Some(reply_to_address).into(),
             sink: Some(method).into(),
-            ttl: Some(ttl),
-            token: options.token().map(|s| s.to_string()),
+            ttl: Some(options.ttl),
```

Comment:
can you also update the doc comment accordingly?

Reply:
yes... wonder when I'll begin to remember to do this without prompting...